src/gui/ConditionalRotationPanel.py

```
@@ -945,9 +945,12 @@ def __init__(self, parent=None, config=None):
         self._load()
 
     def _setup_ui(self):
-        layout = QVBoxLayout(self)
-        layout.setContentsMargins(0, 0, 0, 0)
-        layout.setSpacing(0)
+        # 与 LabelAnd* 控件保持一致的约定：把主布局存为 self.layout 属性（遮蔽 Qt 的
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)